pkg/trace/sampler/sampler.go

```diff
@@ -53,6 +53,10 @@ const (
 // SamplingPriority is the type encoding a priority sampling decision.
 type SamplingPriority int8
 
+// SamplingMechanism this type is an alias for pkg/proto/datadog/trace/remote_rates.proto
+// TargetTPS.mechanism
+type SamplingMechanism uint32
```

Comment:
Would it make sense to keep this definition together with TargetTPS as it used to be originally?